mta-sts.md

```diff
@@ -198,32 +196,80 @@ This JSON object contains the following key/value pairs:
-  ".example.net"]` indicates that mail for this domain might be handled by any
+  receiving mail for this domain.  For example:
+  ```
+   mx: mail.example.com
+   mx: .example.net
+  ```
+  indicates that mail for this domain might be handled by any
   MX with a certificate valid for a host at `mail.example.com` or `example.net`.
   Valid patterns can be either fully specified names (`example.com`) or suffixes
   (`.example.net`) matching the right-hand parts of a server's identity; the
-  latter case are distinguished by a leading period.  In the case of
-  Internationalized Domain Names ([@!RFC5891]), the MX MUST specify the
-  Punycode-encoded A-label [@!RFC3492] and not the Unicode-encoded U-label. The
-  full semantics of certificate validation are described in
-  (#mx-certificate-validation), "MX Certificate Validation."
+  latter case are distinguished by a leading period.  If there are more than
+  one MX specified by the policy, they MUST be on separate lines within the
+  policy file.  In the case of Internationalized Domain Names ([@!RFC5891]), 
+  the MX MUST specify the Punycode-encoded A-label [@!RFC3492] and not the 
+  Unicode-encoded U-label. The full semantics of certificate validation are 
+  described in (#mx-certificate-validation), "MX Certificate Validation."
 
-An example JSON policy is as below:
+An example policy is as below:
 
 ```
-{
-  "version": "STSv1",
-  "mode": "enforce",
-  "mx": [".mail.example.com"],
-  "max_age": 123456
-}
+version: STSv1
+mode: enforce
+mx: mail.example.com
+mx: .example.net
+mx: backupmx.example.com
+max_age: 123456
 ```
 
+The formal definition of the policy resource, defined using [@!RFC7405], is as
+follows:
+
+    sts-policy-record        = sts-policy-version line-delim
+                               sts-policy-mode line-delim
+                               sts-policy-mx line-delim [sts-policy-mx line-delim]
+                               sts-policy-max-age
+
+    line-delim               = %x0d %x0A                          ; "CRLF"
+
+    field-delim              = ":" WSP*
+
+    sts-policy-version       = sts-policy-version-field field-delim
+                               sts-policy-version-value
+
+    sts-policy-version-field = %s"version"
+
+    sts-policy-version-value = %s"STSv1"
+
+    sts-policy-mode          = sts-policy-mode-field field-delim
+                               sts-policy-mode-value
+
+    sts-policy-mode-field    = %s"mode"
+
+    sts-policy-model-value   =  %s"report" / %s"enforce"
+
+    sts-policy-mx            = sts-policy-mx-field field-delim
+                               sts-policy-mx-value
+
+    sts-policy-mx-field      = %s"mx"
+
+    sts-policy-mx-value      = 1*(ALPHA / DIGIT / "_" / "-" / "~" / ".")
+
+    sts-policy-max-age       = sts-policy-max-age-field field-delim
+                               sts-policy-max-age-value
+
+    sts-policy-max-age-field = %s"max_age"
+
+    sts-policy-max-age-value = 1*10DIGIT
```

Comment:
I think this is valid as-is, no? https://tools.ietf.org/html/rfc5234#section-3.7